maptivist/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -206,10 +105,11 @@ public Entity getEntity(DatastoreService datastore, String title) {
         return marker;
     }
 
-    public void updateFlags(Entity markerEntity, DatastoreService datastore, Key markerKey, String newFlag) {
+    // I removed the flag and votes update methods since the votes (and 
```

Comment:
You didn't finish the sentence?

Also, I would delete the code instead of just commenting it out - unless you were going to put the code back very soon.

Reply:
Yeah I'll just delete it. Adetunji said that since he made a voter servlet so that code isn't necessary.